From 3d3364221c235c733436abaeeea9b5e6813b06ae Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 23 Aug 2022 10:27:21 -0700
Subject: [PATCH] Include time.h for time_t

Fixes
src/core/Statistics.hpp:41:37: error: 'time_t' has not been declared
|    41 |                                     time_t last_updated,
|       |                                     ^~~~~~

Upstream-Status: Submitted [https://github.com/ccache/ccache/pull/1145]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/core/Statistics.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/Statistics.hpp b/src/core/Statistics.hpp
index 3e9ed816..0017f209 100644
--- a/src/core/Statistics.hpp
+++ b/src/core/Statistics.hpp
@@ -20,6 +20,7 @@
 
 #include <core/StatisticsCounters.hpp>
 
+#include <ctime>
 #include <string>
 #include <unordered_map>
 #include <vector>
-- 
2.37.2

